### ISSUE TITLE
Bodymovin does not display special unicode characters correctly when building text.

### ISSUE THREAD

**Issue by Danjuanlab** [airbnb/lottie-web#1703]
In recent projects, there is a piece of text with a mask path and an emoji symbol（🐻）, but after exporting, the emoji will be displayed as �� .
![bug1](https://user-images.githubusercontent.com/42509669/60634262-efd01d80-9e40-11e9-9fa9-0bf9141102ef.png)
![bug2](https://user-images.githubusercontent.com/42509669/60634269-f2cb0e00-9e40-11e9-9059-0b1f383b0fdf.png)

Then I tried to find the reason for this problem on Google and found that emoji is the same as the symbols of some languages. It has two or more characters combined. So '🐻'.length===2. When the text has a mask path, the build text is displayed as a single text as a text, and in the source is used **for(i=0;i<str.length ;i++)** 
This Iteration method, so when there is emoji or special text language symbols in this text, it will cause display error, because the length can not correctly divide the text visual character length.
But this problem is actually a big pit of JavaScript processing unicode characters, so I found some related information and found that after ECMAScript 6, the method can be Iteration correctly and quickly by **for...of** , and it can also be correct identify single unit emoji symbols. So the segmentation of characters can be written like this.
```
var st=’here is a 🐻 with mask path‘；
for (const s of st){
          console.log(s);
}
--log:["h", "e", "r", "e", " ", "i", "s", " ", "a", " ", "🐻", " ", "w", "i", "t", "h", " ", "m", "a", "s", "k", " ", "p", "a", "t", "h"]
```
But since I don't know how this modification will bring  other problems besides being incompatible with JavaScript below ECMAScript 6. At the same time, I am not very clear about the structure of the bodymovin's build function, so I hope you can refer to the contents of this post to try to fix this bug.

[javascript-unicode reference link](https://mathiasbynens.be/notes/javascript-unicode)
[text unicode bug.zip](https://github.com/airbnb/lottie-web/files/3357304/text.unicode.bug.zip)

**Comment by Danjuanlab** [comment]
![无标题3](https://user-images.githubusercontent.com/42509669/60646991-3ccae880-9e6f-11e9-82ac-f054b5063ff4.png)
```
TextProperty.prototype.buildFinalText = function(text) {
    var combinedCharacters = FontManager.getCombinedCharacterCodes();
    var charactersArray = [];
    for (const s of text){
    	charactersArray.push(s);
    }
    return charactersArray;
}
```
Substituting this code for the original buildFinalText function will allow the text with the mask path to split the string correctly.  But I don't understand this condition a bit, so I think you should be able to write this code more complete.


```
 if (combinedCharacters.indexOf(text.charCodeAt(i)) !== -1) {
             charactersArray[charactersArray.length - 1] += text.charAt(i);
}
```

**Comment by bodymovin** [comment]
Hey thanks for the link and the research!
I've fixed it and should be working on version 5.5.6
